docs/code/modules.rst

```diff
@@ -204,3 +214,17 @@ Networks
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .. autoclass:: texar.modules.Conv1DNetwork
     :members: default_hparams,forward,append_layer,has_layer,layer_by_name,layers_by_name,layers,layer_names
+
+Bert
+=========
+
+.. spelling::
+    pretrained
+    pooler
+    hparams
+    bert
```

Comment:
- The correct spelling for "*pretrained*" is actually "*pre-trained*".
- "*hparams*" should not be a word -- either use "*hyperparameters*" or `` `hparams` ``.
- Similarly, use "*BERT*" instead of "*bert*". The former would probably not trigger a warning because it is considered an acronym.